examples/heartbeat-print.cpp

```diff
@@ -63,7 +63,7 @@ int main(int argc, char *argv[])
     struct sockaddr_in sockaddr;
     const socklen_t addrlen = sizeof(sockaddr);
     int port = 14550;
-    int fd;
+    int fd = 0;
```

Comment:
this is nonsense and hides bug that would rather be caught by valgrind

Reply:
Changed back and added a comment about it.